pkg/kubelet/cadvisor/types.go

```diff
@@ -44,4 +44,8 @@ type Interface interface {
 
 	// HasDedicatedImageFs returns true iff a dedicated image filesystem exists for storing images.
 	HasDedicatedImageFs() (bool, error)
+
+	// GetFsInfoByFsUUID returns the stats of the filesystem with the specified
+	// uuid.
+	GetFsInfoByFsUUID(uuid string) (cadvisorapiv2.FsInfo, error)
```

Comment:
adding this without implementing in _unsupported and _windows broke cross-build. fix in https://github.com/kubernetes/kubernetes/pull/52073

Reply:
Thanks. Forgot about the cross-build. It would be nice if this can be caught in presubmit test.